app/crypto/EncryptedValue.scala

```diff
@@ -39,7 +39,7 @@ class AesGCMCrypto @Inject()() {
 
   private val IV_SIZE = 96
   private val TAG_BIT_LENGTH = 128
-  val ALGORITHM_TO_TRANSFORM_STRING = "AES/GCM/PKCS5Padding"
+  val ALGORITHM_TO_TRANSFORM_STRING = "AES/GCM/NoPadding"
```

Comment:
is this change required.. given we are only upgrading the jvm version

Reply:
Platops say it's fine, won't cause issues.